tests/functional/pv/pv_services/test_cephfs_pvc_with_chunk_size.py

```diff
@@ -0,0 +1,96 @@
+import time
+import pytest
+import logging
+from ocs_ci.framework.testlib import bugzilla, tier2
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import cluster
+from ocs_ci.framework.pytest_customization.marks import magenta_squad
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    argnames="interface",
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHFILESYSTEM], marks=pytest.mark.polarion_id("OCS-323")
+        ),
+    ],
+)
```

Comment:
do you really need parameterize here? You are running it only for cephfs

Reply:
removed parametrize